### ISSUE TITLE
Добавить опцию контакт-листа - скрывать dummy-контакты.

### ISSUE THREAD

**Issue by SM-Sergey** [miranda-ng/miranda-ng#4807]
Вот сюда добавить еще одну опцию - Hide Dummy contacts (скрывать контакты, обслуживаемые протоколом "Dummy")

![2024-12-16 23_09_31-Miranda NG options](https://github.com/user-attachments/assets/df4b9bfa-bf63-41db-96a6-9f7788f1fc18)

И сюда тоже - кнопку на тулбар "Show/hide dummy contacts"

![2024-12-16 23_12_58-Miranda NG options](https://github.com/user-attachments/assets/062b32b0-8791-4b3f-8654-55fa35b96c15)

**Comment by georgehazan** [comment]
Есть мнение, что проще прогуляться один раз по этим контактам и скрыть их (возможно, для этого потребуется поставить плагин MenuItemEx)

**Comment by SM-Sergey** [comment]
Ну нет же, их надо оперативно скрывать-раскрывать. Их больше двух сотен, и десятка три из них требуют периодического обращения в их историю. Поэтому нужна кнопка на тулбаре. Я ж не просто так открываю тикеты.
Неужели это так мегасложно???

PS
Группы  тоже не катят, они используются в игнорах для быстрого включения-выключения игноров группам, а они оказываются полностью несовместимы с желаемой группировкой в клисте, поэтому в клисте они отключены раз и навсегда, ибо в игнорах они важнее и нужнее.

**Comment by SM-Sergey** [comment]
Верните пожалуйста мою просьбу! Пусть она не самая срочная, но без этого очень неудобно!

**Comment by georgehazan** [comment]
> Ну нет же, их надо оперативно скрывать-раскрывать.

Для этого есть группы, которые можно оперативно скрывать-раскрывать. Не надо добавлять лишние сущности

**Comment by georgehazan** [comment]
> Группы тоже не катят, они используются в игнорах для быстрого включения-выключения игноров группам

Может всё же использовать группы для того, зачем они изначально предполагались?

**Comment by SM-Sergey** [comment]
Нет. Группы для включения-выключения игноров мне (и наверняка не только мне) нужнее. Никак от этого не отказаться. Это важнее, чем куча думми контактов, вечно болтающихся в клисте. Соответственно, включение групп в клисте не обсуждается в принципе. Да, они скрываются, конечно, вместе с оффлайн контактами, но это не всегда удобно, бывает надо писать в оффлайны живых протоколов. Поэтому нужно отдельное скрытие тех и этих.

Ну я не верю ни разу, что такая кнопка в тулбаре - это прямо экстра-сложно. Небось пять минут, несколько строчек кода и иконка. Зато сильное облегчение жизни и немалая прибавка удобства.
